dev/app/auth/tpl/error.jade

```diff
@@ -0,0 +1 @@
+h1 failed to log in
```

Comment:
This is error.jade, it can come due to anything, so why set it is 'failed to login'?